README.md

```diff
@@ -6,7 +6,7 @@ Poli-Tech AI Capstone Project
 | Date     | Revision | Description         | Author                          |
 | -------- | -------- | ------------------- | ------------------------------- |
 | mm/dd/yy | x.x      | Version Description | Name(s)                         |
-| 01/23/20 | 0.1      | Initial version     | Jacob, David, Alex, Eric        |
+| 04/30/20 | 0.1      | Initial version     | Jacob, David, Alex, Eric        |
```

Comment:
This should probably be updated to Date - June 23rd, Revision - 1.0, Description - First deployment